packages/turf-point-grid/index.d.ts

```diff
@@ -3,6 +3,6 @@ import {BBox, Points, Units, Feature, Features} from '@turf/helpers';
 /**
  * http://turfjs.org/docs/#pointgrid
  */
-declare function pointGrid(bbox: BBox | Feature<any> | Features<any>, cellSide: number, units?: Units, centered?: boolean, bboxIsMask?: boolean): Points;
+declare function pointGrid(bbox: BBox | Feature<any> | Array<number>, cellSide: number, units?: Units, centered?: boolean, bboxIsMask?: boolean): Points;
```

Comment:
That would be a good idea, maybe I'll create that before updating all the definitions.

My approach to Typescript is not as strict as others like [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) which enforce strict linting (don't think we are there yet for TurfJS - might add a basic TS linting config).